README.md

```diff
@@ -234,7 +234,21 @@ Returns a paged list of all permissions currently associated with this vehicle.
 #### Return
 | Type               | Description         |
 |:------------------ |:--------------------|
-| List[String]       | 	An array of permissions. |
+| String[]       | 	An array of permissions. |
+
+### `has_permissions(self, permissions)`
+
+Returns if vehicle contains specified permission(s).
```

Comment:
sorry this is extremely nit but this makes it sound like it only returns sometimes, can we do `Returns whether the vehicle...`